src/Components/Common/Header/Header.js

```diff
@@ -14,7 +14,7 @@ import MoreVert from '@material-ui/icons/MoreVert'
 import ViewList from '@material-ui/icons/ViewList'
 
 // core components
-import Button from '../CustomButtons/Button.js'
+import Button from '../UI/CustomButtons/Button.js'
```

Comment:
The `CustomButtons/Button` is the template button, can you use the template button and delete the existing in the UI folder?